datumaro/plugins/yolo_format/converter.py

```diff
@@ -164,12 +248,148 @@ def patch(cls, dataset, patch, save_dir, **kwargs):
+    def _make_annotation_line(self, width: int, height: int, anno: Annotation) -> Optional[str]:
+        if anno.label is None or not isinstance(anno, Polygon):
+            return
+        values = [value / size for value, size in zip(anno.points, cycle((width, height)))]
+        string_values = " ".join("%.6f" % p for p in values)
+        return "%s %s\n" % (anno.label, string_values)
+
+
+class Yolo8OrientedBoxesConverter(Yolo8Converter):
+    def _make_annotation_line(self, width: int, height: int, anno: Annotation) -> Optional[str]:
+        if anno.label is None or not isinstance(anno, Bbox):
+            return
+        points = _bbox_annotation_as_polygon(anno)
+        values = [value / size for value, size in zip(points, cycle((width, height)))]
+        string_values = " ".join("%.6f" % p for p in values)
+        return "%s %s\n" % (anno.label, string_values)
+
+
+class Yolo8PoseConverter(Yolo8Converter):
+    @cached_property
+    def _map_labels_for_save(self):
+        point_categories = self._extractor.categories().get(
+            AnnotationType.points, PointsCategories.from_iterable([])
+        )
+        return {label_id: index for index, label_id in enumerate(sorted(point_categories.items))}
+
+    def _save_config_files(self, subset_lists: Dict[str, str]):
+        extractor = self._extractor
+        save_dir = self._save_dir
+
+        point_categories = extractor.categories().get(
+            AnnotationType.points, PointsCategories.from_iterable([])
+        )
+        if len(set(len(cat.labels) for cat in point_categories.items.values())) > 1:
+            raise DatasetExportError(
+                "Can't export: skeletons should have the same number of points"
+            )
+        n_of_points = (
+            len(next(iter(point_categories.items.values())).labels)
+            if len(point_categories) > 0
+            else 0
+        )
+
+        with open(osp.join(save_dir, self._config_filename), "w", encoding="utf-8") as f:
+            label_categories = extractor.categories()[AnnotationType.label]
+            parent_categories = {
+                self._map_labels_for_save[label_id]: label_categories.items[label_id].name
+                for label_id in point_categories.items
+            }
+            assert set(parent_categories.keys()) == set(range(len(parent_categories)))
+            data = dict(
+                path=".",
+                names=parent_categories,
+                kpt_shape=[n_of_points, 3],
+                **subset_lists,
+            )
+            yaml.dump(data, f)
+
+    def _make_annotation_line(self, width: int, height: int, skeleton: Annotation) -> Optional[str]:
+        if skeleton.label is None or not isinstance(skeleton, Skeleton):
+            return
+
+        x, y, w, h = skeleton.get_bbox()
+        bbox_values = _make_yolo_bbox((width, height), [x, y, x + w, y + h])
+        bbox_string_values = " ".join("%.6f" % p for p in bbox_values)
```

Comment:
Do we really need such a high precision? In most cases, 2 digits are enough. I suggest that we print at most 2 decimal digits.